apps/src/storage/dataBrowser/DataLibraryPane.jsx

```diff
@@ -66,4 +83,11 @@ class DataLibraryPane extends React.Component {
   }
 }
 
-export default Radium(DataLibraryPane);
+export default connect(
+  state => ({}),
+  dispatch => ({
+    onShowWarning(warningMsg, warningTitle) {
+      dispatch(showWarning(warningMsg, warningTitle));
```

Comment:
Would you mind adding a link to where showWarning is implemented to help with reviewing? (can just go in CR comments... not needed in the code)

Reply:
Opening and closing is controlled by redux here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/storage/redux/data.js#L77

The dialog itself is here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/storage/dataBrowser/DataWorkspace.jsx#L96

Reply:
Looks good. Thank you!